composer.json

```diff
@@ -1,9 +1,11 @@
 {
     "name": "sonata-project/twig-extensions",
-    "type": "symfony-bundle",
-    "description": "Symfony twig extensions",
+    "type": "library",
```

Comment:
not sure about this, don't merge.

Reply:
It is a lib AFAIK

Reply:
yes, I just didn't know if I was going to make it a php lib or a symfony bundle, at the end it is lib with a Symfony Bridge, so this is ok.